packaging/rpm-ostree.spec.in

```diff
@@ -129,6 +124,12 @@ Requires:       librepo%{?_isa} >= %{librepo_version}
 #                     end of libdnf build deps                          #
 #########################################################################
 
+%if 0%{?rhel} <= 8
```

Comment:
In theory we could probably just drop rhel 8 stuff from this file now...

Reply:
Indeed, can do that. Are we OK to do this in a follow-up to not waste CI?

Reply:
https://github.com/coreos/rpm-ostree/pull/4156